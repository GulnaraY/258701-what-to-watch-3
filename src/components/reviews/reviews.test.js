import React from 'react';
import rerender from 'react-test-renderer';
import Reviews from './reviews.jsx';

const reviews = new Array(3).fill(``).map(() => (
  {
    author: `Tim Cook`,
    text: `hello`,
    rating: 2.0,
    dateTime: `10 26 2019`,
  }
));

it(`Should render Reviews`, () => {
  const tree = rerender.create(
      <Reviews
        reviews={reviews}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
