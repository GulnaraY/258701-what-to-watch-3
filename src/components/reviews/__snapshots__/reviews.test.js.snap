// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Reviews 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          hello
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Tim Cook
          </cite>
          <time
            className="review__date"
            dateTime="10 26 2019"
          >
            10 26 2019
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        2
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          hello
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Tim Cook
          </cite>
          <time
            className="review__date"
            dateTime="10 26 2019"
          >
            10 26 2019
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        2
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          hello
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Tim Cook
          </cite>
          <time
            className="review__date"
            dateTime="10 26 2019"
          >
            10 26 2019
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        2
      </div>
    </div>
  </div>
</div>
`;
